tests/drivers/nrf_wifi/raw_tx_rx/prj.conf

```diff
@@ -0,0 +1,66 @@
+# Copyright (c) 2025 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+CONFIG_ZTEST=y
+CONFIG_ENTROPY_GENERATOR=y
+CONFIG_TEST_RANDOM_GENERATOR=y
+CONFIG_WIFI=y
+CONFIG_WIFI_NRF70=y
```

Comment:
not sure why options are being set here that are completely ignored because sysbuild controls them